src/icons.ts

```diff
@@ -104,7 +105,7 @@ export const icons: Icon[] = [
   i("n8n", { [c]: "n8n.svg" }),
   i("NextJS", { [c]: "nestjs.svg" }),
   i("NocoDB", { [c]: "nocodb.png" }),
-  i("NodeJS", { [c]: "nodejs.svg" }),
+  i("NodeJS", { [c]: "nodejs.svg", [d]: "nodejs-black.svg", [l]: "nodejs-white.svg" }),
```

Comment:
Can we change the naming to `dark` and `light` instead of `black` and `white` for consistency?